src/ctap/mod.rs

```diff
@@ -918,21 +919,31 @@ impl CtapState {
         let mut signature_data = auth_data.clone();
         signature_data.extend(client_data_hash);
 
-        let (signature, x5c) = if env.customization().use_batch_attestation() || ep_att {
-            let attestation_private_key = storage::attestation_private_key(env)?
-                .ok_or(Ctap2StatusCode::CTAP2_ERR_VENDOR_INTERNAL_ERROR)?;
-            let attestation_key =
-                crypto::ecdsa::SecKey::from_bytes(&attestation_private_key).unwrap();
-            let attestation_certificate = storage::attestation_certificate(env)?
-                .ok_or(Ctap2StatusCode::CTAP2_ERR_VENDOR_INTERNAL_ERROR)?;
-            (
-                attestation_key
-                    .sign_rfc6979::<Sha256>(&signature_data)
-                    .to_asn1_der(),
-                Some(vec![attestation_certificate]),
-            )
+        let attestation_id = if env.customization().use_batch_attestation() {
+            Some(attestation_store::Id::Batch)
+        } else if ep_att {
+            Some(attestation_store::Id::Enterprise)
         } else {
-            (private_key.sign_and_encode(env, &signature_data)?, None)
+            None
+        };
```

Comment:
Just noticed that the order is wrong. Enterprise attestation has priority. For all other occasions, it depends on the security key's customization.

Reply:
Done.